package.json

```diff
@@ -1,12 +1,13 @@
 {
-  "name": "hubot-stackstorm",
-  "description": "A hubot plugin for integrating with StackStorm event-driven infrastructure automation platform.",
-  "version": "0.4.5",
-  "author": "StackStorm, Inc. <info@stackstorm.com>",
+  "name": "hubot-stackstorm-auth",
```

Comment:
What is going on in this file? :)

Did you intend to open this PR here or your fork?